paper.tex

```diff
@@ -634,112 +639,90 @@ \section*{Genome ARGs}
+% (or that all internal nodes are either coalescences or recombinations)
+% is when we are incorporating ancient genomes into ARG
+% inference~\citep{speidel2021inferring,wohns2022unified}.
+% Similarly, a gARG may contain nodes that
+% do not correspond to any particular event in the ancestry of the samples.
+% For example, node \textsf{h} is the direct ancestor of \textsf{d} in
+% Fig~\ref{fig-arg-in-pedigree}, and is not the product of either a
+% coalescence or a recombination. Such nodes would usually be removed
+% from the graph so that \textsf{d} descends directly from \textsf{k} (see
+% the ``\nameref{ARG_simplification}'' section) but there is no necessity for this from
+% a representational perspective.
+% Secondly, because we do not classify nodes by event type
+% (common ancestor or recombination) there is no limitation
+% on the complexity that can be modelled---multiple recombinations
+% and coalescences can happen simultaneously in the same genome,
+% a common occurance in large deeply sequenced
+% pedigrees~\citep{hayes20191000,RosFreixedes2022}.
+% Finally, there is a subtle and important point about how recombination
+% is represented: which genome in the organismal lifecycle
+% do nodes represent?
+% In principle, we can interpret nodes as representing genomes
+% at any point in the life cycle [supplementary Figure ?]. Hudson
+% views nodes as representing gametes~\citep{hudson1983properties}.
+% However, we argue that the most intuitive
+% approach is to let nodes represent the genomes present in diploid
+% individuals in each generation, as in
+% Fig~\ref{fig-arg-in-pedigree}A. In this case recombination
+% is not represented by a single ``recombination node'' (as in an eARG), but with
+% two parent genomes between which recombination takes place
+% (e.g.\ nodes \textsf{e} and \textsf{f}), and a
+% single resulting ``recombinant'' genome (e.g.\ \textsf{c}). The
+% edges that link this recombinant with its two parents carry the information
+% concerning breakpoint location that would be associated with a
+% recombination node in an eARG. [TODO finish up here with more
+% clarity on what this means in terms of the required specifity
+% of recombination events, and put in a forward ref to the simplification
+% section where we talk about stacking up recombination events
+% and their identifiability.]
+
+The eARG and gARG definitions provided here are apparently very
+similar, but, as we shall see, there are many advantages to decoupling
+our representation of patterns of genetic inheritance
+from the events that generated them.
 For simplicity, throughout the rest of the paper we will contrast the
-classical eARG (as defined in the XXX section) with the gARG definition
-here. However, it is important to note that many of the benefits of the
+classical eARG with the gARG definition here.
+However, it is important to note that many of the benefits of the
 gARG encoding that we discuss are due to the encoding of inheritance via
```

Comment:
I think we wanted to reserve "encoding" for a different purpose, and refer to "annotation" of nodes vs edges, right?

Reply:
So more like:
> However, it is important to note that many of the benefits of the gARG
encoding that we discuss are due to tracing paths of inheritance by annotating
edges with intervals rather than annotating nodes with breakpoints.

Reply:
Yes, we need to get at something like this. I'll have another go at this final para.